mock/cpp_mock_scenarios/src/follow_lane/acquire_position_in_world_frame.cpp

```diff
@@ -30,9 +29,7 @@ class DecelerateAndFollowScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit DecelerateAndFollowScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "idiot_npc", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.